file-formats/ints/type/zif_aff_ints_v1.intf.abap

```diff
@@ -0,0 +1,322 @@
+      "! This is the value of an additional information
+      value               TYPE c LENGTH 1024,
+      "! <p class="shorttext">Additional Information Position ID</p>
+      "! This is the position of an additional information ID
+      position_id         TYPE i,
+    END OF ty_additional_info,
+
+    "! <p class="shorttext">Additional Information Texts</p>
+    "! These are the scenario additional information texts
+    BEGIN OF ty_additional_info_text,
+      "! <p class="shorttext">Language Key</p>
+      "! This is the language key
+      language            TYPE sy-langu,
+      "! <p class="shorttext">Additional Information Key</p>
+      "! This is an additional information key
+      additional_info_key TYPE c LENGTH 50,
+      "! <p class="shorttext">Additional Information Value ID</p>
+      "! This is the additional information value ID
+      value_id            TYPE i,
+      "! <p class="shorttext">Additional Information Key Text Label</p>
+      "! This is the additional information key text label
+      key_label           TYPE c LENGTH 255,
+      "! <p class="shorttext">Additional Information Value Text Label</p>
+      "! This is the additional information value text label
+      value_label         TYPE c LENGTH 255,
+    END OF ty_additional_info_text,
+
+    "! <p class="shorttext">SBS Segment Fields</p>
+    "! These are the SBS Scenario Segment Fields
+    BEGIN OF ty_sbs_segment_fields,
+      "! <p class="shorttext">Table or View Field Name</p>
+      "! This is the field name of a table or view
+      fieldname      TYPE c LENGTH 30,
+      "! <p class="shorttext">Position</p>
+      "! This is the position
+      field_position TYPE i,
+      "! <p class="shorttext">Key Field</p>
+      "! This is key field or not
+      iskey          TYPE c LENGTH 1,
+    END OF ty_sbs_segment_fields,
+
+    "! <p class="shorttext">Intelligent Scenario</p>
+    "! This is an intelligent scenario
+    BEGIN OF ty_main,
+      "! $required
+      format_version            TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                    TYPE zif_aff_types_v1=>ty_header_60,
+      "! <p class="shorttext">Intelligent Scenario</p>
+      "! This is an intelligent scenario
+      "! $required
+      intelligent_scenario      TYPE ty_intelligent_scenario,
+      "! <p class="shorttext">Objects Based on Scenario</p>
+      "! These are the objects based on scenario
+      scenario_ddl_objects      TYPE STANDARD TABLE OF ty_scenario_ddl_objects WITH DEFAULT KEY,
+      "! <p class="shorttext">Binding Mappings</p>
+      "! These are scenario binding mappings
+      binding_mappings          TYPE STANDARD TABLE OF ty_binding_mappings WITH DEFAULT KEY,
+      "! <p class="shorttext">Dataset Bindings</p>
+      "! These are scenario dataset bindings
+      bindings                  TYPE STANDARD TABLE OF ty_bindings WITH DEFAULT KEY,
```

Comment:
I am not sure how `bindings` and `binding_mappings` belong together. Would it make sense to combine these?

Similar question for `signature*`, `modeling*` and `parameter*`.

Reply:
I removed bindings mappings as we decided to make it obsolete. So, we have only bindings now.
Merged the structures signature* & parameter*.
And also for current scope these structures will be empty as it's relevant for other scenario types apart from GenAI and we are targeting only GenAI for 2502.